src/main/kotlin/kotliquery/Row.kt

```diff
@@ -241,30 +241,38 @@ data class Row(
     fun zonedDateTime(columnIndex: Int): ZonedDateTime {
         return zonedDateTimeOrNull(columnIndex)!!
     }
-    fun zonedDateTimeOrNull(columnIndex: Int): ZonedDateTime? {
-        return nullable(ZonedDateTime.ofInstant(sqlTimestampOrNull(columnIndex)?.toInstant(), ZoneId.systemDefault()))
-    }
+    fun zonedDateTimeOrNull(columnIndex: Int) = nullable(
```

Comment:
Please don't remove the declaration of return types here.

Reply:
Return types are back :)